manage_breast_screening/config/jinja2_env.py

```diff
@@ -34,22 +69,61 @@ def format_words(value, separator="_"):
         # - the whole thing is upper case
         # - there is an upper case character after the first letter
         if part == part.upper() or len(part) >= 2 and (part[1:].lower() != part[1:]):
+            # Use the acronym as is
             result.append(part)
         else:
+            # lowercase the word
             result.append(part.lower())
 
     return " ".join(parts)
 
 
 def format_date(value):
+    """
+    Format a date
+
+    >>> from datetime import date
+    >>> format_date(date(2025, 1, 1))
+    '1 January 2025'
+    """
     return value.strftime("%-d %B %Y")
 
 
+def format_relative_date(value):
+    """
+    Format a date relative to today
+    """
+    return arrow.get(value).humanize(granularity="day")
+
```

Comment:
I seem to recall having to do something slightly custom for this because the way it counted days ended up off by one than what might be intuitive. Would be good to check this does things correctly.

Reply:
@edwardhorsford yep, this was wrong as well. I've reimplemented it and added some tests, so it should be consistent with yours now 🤞🏻